ddpui/api/dbt_api.py

```
@@ -164,128 +164,27 @@ def put_dbt_github(request, payload: OrgDbtGitHub):
-    except Exception as err:
-        logger.error(f"Failed to sync .gitignore contents: {err}")
-        raise HttpError(500, f"Failed to sync .gitignore contents: {err}") from err
-
-    return {
-        "success": True,
-        "gitrepo_url": payload.gitrepoUrl,
-        "message": "Successfully connected to remote git repository",
-    }
+            return dbt_service.connect_git_remote(orguser, payload)
+    except Exception as e:
+        logger.error(f"Git remote connection failed: {str(e)}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)